lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
```

Comment:
I wasn't aware of `createPseudo` api, and it seems that it removes the need for referencing `markFunction`.

However, we still need to move `runescape, funescape, getText`, to ours, which would require some duplicate codes.

One way would be to get rid of original implementation (in such case we can remove `getText` declaration) and use our own string unescaping procedure and `getText`. If we choose this way, we can make string escaping procedure of `:contains` consistent with that of `:matches-css`. Maybe `getText` is not needed at all, but to be sure I need to check that `innerText` works well in IE 9.